ask-sdk-servlet-support/src/com/amazon/ask/servlet/verifiers/ServerRequest.java

```diff
@@ -0,0 +1,41 @@
+/*
+    Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+    except in compliance with the License. A copy of the License is located at
+
+        http://aws.amazon.com/apache2.0/
+
+    or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+    the specific language governing permissions and limitations under the License.
+ */
+
+package com.amazon.ask.servlet.verifiers;
+
+import com.amazon.ask.model.RequestEnvelope;
+
+/**
+ * Provides container for server request that should be validated
+ */
+public interface ServerRequest {
```

Comment:
Trying to think of a more descriptive name for this..AlexaHttpRequest? But I like the abstraction. We might take this a step further in the future and move this down into the runtime module and make the request type (in this case RequestEnvelope) a generic.

Reply:
Sounds, good. In my case I use only `Verifier` part in my project since I use Spring Cloud Gateway which relies on reactive stack. Copying code into my project seemed awkward, thus this PR was created.